src/content/ui-api/management-api/authentication/index.md

```diff
@@ -0,0 +1,117 @@
+
+```nohighlight
+gs-<installation-name>
+```
+
+This context is selected automatically as the current context, so you should be ready to use kubectl with the management API.
+
+When switching back to this context, it should not be necessary to go through the web-based authentication flow again. `kubectl` will automatically refresh the authentication token when needed, without your interaction.
+
+### Alternative method
+
+You can alternatively initiate the single sign-on authentication directly in a browser, without a need to install the `kubectl` plug-in.
+
+We provide a login helper utility. If you have your installation's management API endpoint URL, you can construct the utility's URL by prepending `login.` to the host name.
+
+If, for example, your management API URL is
+
+```nohighlight
+https://g8s.base.domain.tld
+```
+
+then the login utility can be accessed via
+
+```nohighlight
+https://login.g8s.base.domain.tld
+```
+
+The tool will immediately redirect you to your identity provider's authentication flow where you proceed providing your credentials as usual. After that, or if you are already authenticated in the current browser, you will be redirected to a resulting page.
+
+The screenshot shows an example of that result page.
+
+![Login helper screenshot](login-utility-results.png)
+
+Here you can inspect the details that will be passed to the management API as part if the ID token. You can use this to verify the details coming from your identity provider, especially the `email` (which is used as your user identifier) and `groups` claim.
+
+The rest of the page helps you set up `kubectl` manually.
+
+## Authenticating for programmatic access {#service-auth}
+
+For programmatic access, for example from CI/CD pipelines, you should not rely on the above authentication mechanism. Instead, please use [service accounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/).
+
+Each Giant Swarm installation provides a service account named `automation` in the `default` namespace. The according token can be found by inspecting this `ServiceAccount` resource. It is usually named starting with `automation-token-`.
+
+**Note:** Since the above token is extremely powerful, **we strongly recommend to create a specific service account for each integration**, binding it to specific roles and only to the required namespaces.
+
+## Single sign-on requirements {#sso-requirements}
+
+Giant Swarm assists all customers with setting up single sign-on (SSO) for the management API. During that process, we make sure that these requirements are met:
+
+- As a customer, you need to decide for an **identity provider** to use. Most enterprise-grade organizations have their solution in place. Since we use [Dex](https://github.com/dexidp/dex) as a connector between the management API and your identity provider, we can support a variety of common standards like OpenID Connect (OIDC) and LDAP.
```

Comment:
```suggestion
- As a customer, you need to decide on an **identity provider** to use. Most enterprise-grade organizations have a solution already in place. Since we use [Dex](https://github.com/dexidp/dex) as a connector between the management API and your identity provider, we can support a variety of common standards like OpenID Connect (OIDC) and LDAP.
```

Reply:
slight, potentially v. british thought - by saying that enterprise-grade orgs have a provider already, we imply that orgs reading this that don't aren't 'enterprise-grade', which might rub a few people the wrong way

Reply:
Get it. Will modify.